docs/docs/recipes.md

```diff
@@ -130,7 +130,47 @@ Data sourcing in Gatsby is plugin-driven; Source plugins fetch data from their s
 
 ## Transforming data
 
-Transforming data in Gatsby is also plugin-driven; Transformer plugins take data fetched using source plugins, and process it into something more usable (e.g. JSON into JavaScript objects, markdown to HTML, and more).
+Transforming data in Gatsby is plugin-driven. Transformer plugins take data fetched using source plugins, and process it into something more usable (e.g. JSON into JavaScript objects, markdown to HTML, and more). `gatsby-transformer-plugin` can transform your markdown files to HTML.
 
-- Walk through an example using the `gatsby-transformer-remark` plugin to transform markdown files [tutorial part six](/tutorial/part-six/#transformer-plugins)
-- Search available transformer plugins in the [Gatsby library](/plugins/?=transformer)
+### Prerequisites
+
+- A Gatsby site with `gatsby-config.js` and an `index.js` page
+- A Markdown file saved in your Gatsby site `src` directory
+- The `gatsby-transformer-remark` plugin installed
+
+### Directions
+
+1. Add the plugin in your `gatsby-config.js` page:
+
+```
+// In your gatsby-config.js
+plugins: [`gatsby-transformer-remark`],
```

Comment:
i would not place this plugin alone here. a note about `gatsby-source-filesystem` would be good to avoid some pitfalls. see also the PR #15604 for Issue #15594

Reply:
Since that issue and PR are specifically for MDX I'm not sure that's relevant to this recipe is it? If this plugin works in its simplest configuration (which I imagine it does) then I think keeping it this way is still okay.

Reply:
no - all transformer plugin working only if there are nodes available which are imported via a source-plugin.
that was the main pain point in the linked PR

Reply:
at least i would give this hint:

```suggestion
plugins: [
  ... // source plugins to create nodes to transform later
  `gatsby-transformer-remark`,
],
```

Reply:
I think a hint is a good idea–especially if it could save someone a headache when getting started. I added a recommendation to the prerequisites section, and we could also include something like: `not shown: gatsby-source-filesystem for creating nodes to transform`

Reply:
I pushed an update to get this finished up! 👍